### DISCUSSION TITLE
How do I yank an error message?

### DISCUSSION THREAD

**Discussion by David-Else** [helix-editor/helix#3726] [Q&A] [answered]
Often Rust error messages contain solutions that are not available as actions, and also copying them to forums to get help would be fantastic. At the moment when they appear and I try to highlight them to copy with the mouse they just disappear! Is it possible to yank them to the clipboard?

**Comment by Ordoviz** [comment, marked as answer]
In some terminals, such as [foot](https://codeberg.org/dnkl/foot), you can hold <kbd>Shift</kbd> to select text without sending mouse events to Helix.

**Reply by Omnikar** [reply]
I use iTerm2, and it's the ⌥ (option) key here.

**Comment by kirawi** [comment]
I've implemented #3786 so you can yank from the diagnostics picker.
